### PR TITLE
setup unit tests

### PR DESCRIPTION
**Pull request by OmriBarZik**

### CODE REVIEW

File: tests/bootstrap.php [hunk 1]
```diff
@@ -50,3 +50,7 @@ function fix_qunit_html_urls( $html ){
 }
 
 require $_tests_dir . '/includes/bootstrap.php';
+require 'phpunit/local-factory.php';
```
Comment (matipojo) [inline tests/bootstrap.php]: Use  $_tests_dir

File: tests/phpunit/elementor/base/test-bootstrap.php [hunk 2]
```diff
@@ -1,11 +1,16 @@
 <?php
+namespace Elementor\Testing;
 
-class Elementor_Test_Base extends WP_UnitTestCase {
+remove_action( 'admin_init', '_maybe_update_themes' );
```
Comment (matipojo) [inline tests/phpunit/elementor/base/test-bootstrap.php]: move to `setupBeforeClass`

File: tests/phpunit/elementor/test-controls.php [hunk 3]
```diff
@@ -1,29 +1,33 @@
 <?php
+namespace Elementor\Testing;
 
-class Elementor_Test_Controls extends WP_UnitTestCase {
+use  \Elementor\Plugin;
```
Comment (matipojo) [inline tests/phpunit/elementor/test-controls.php]: double space

File: tests/phpunit/elementor/test-controls.php [hunk 4]
```diff
@@ -1,29 +1,33 @@
...
 
 	public function test_getControls() {
-		$this->assertNotEmpty( Elementor\Plugin::$instance->controls_manager->get_controls() );
+		$this->assertNotEmpty( Plugin::$instance->controls_manager->get_controls() );
 	}
 
 	public function test_renderControls() {
 		ob_start();
-		Elementor\Plugin::$instance->controls_manager->render_controls();
+		Plugin::$instance->controls_manager->render_controls();
```
Comment (matipojo) [inline tests/phpunit/elementor/test-controls.php]: Replace `Plugin::$instance` with `$this->plugin`.

File: tests/phpunit/elementor/test-controls.php [hunk 5]
```diff
@@ -81,10 +85,10 @@ public function test_replaceStyleValues() {
 				'right' => '2',
 				'bottom' => '3',
 				'left' => '4',
-			]
+			],
 		];
 
-		$value_callback = function ( $control ) use ( $values ) {
+		$value_callback = function( $control ) use ( $values ) {
```
Comment (matipojo) [inline tests/phpunit/elementor/test-controls.php]: `function( $control )` missing space

**Review by matipojo — CHANGES_REQUESTED** [review]

File: tests/bootstrap.php [hunk 6]
```diff
@@ -23,7 +24,7 @@
 
 tests_add_filter(
 	'muplugins_loaded',
-	function() {
+	function () {
```
Comment (KingYes) [inline tests/bootstrap.php]: ?

**Review by KingYes — CHANGES_REQUESTED** [review]

**Pull request merged**